features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/themes/wso2/templates/label/label-add/template.jag

```diff
@@ -64,7 +65,7 @@
     <div class="form-group">
 		<label class="control-label col-sm-3"><%=i18n.localize("Label")%>  <span class="requiredAstrix">*</span></label>
 		<div class="col-md-9">
-		    <input class="form-control" type="text" id="labelName" name="labelName"/>
+		    <input class="form-control" type="text" id="labelName" name="labelName"  <%=nameFieldDisableStatus%>/>
```

Comment:
variable name `nameFieldDisableStatus ` seems redundant.

Reply:
in the beginning, we set nameFieldDisableStatus to ="" which is allowed to edit label name and if already set label received we set it to read-only to disable editing the label name.